apps/wing/fixtures/valid1/API.md

```diff
@@ -0,0 +1,24 @@
+<h2>API Reference</h2>
+
+<h3>Table of Contents</h3>
+
+- **Classes**
+  - <a href="#valid1.Foo">Foo</a>
+
+<h3 id="valid1.Foo">Foo (preflight class)</h3>
```

Comment:
Using these HTML header tags doesn't feel totally right, but it was the only way I could figure out to get the anchors working as expected.